src/reducers/comments.js

```diff
@@ -3,6 +3,39 @@ import * as types from '../actionTypes/comments';
 
 const { comments } = initialState;
 
+const commentsData = (state = [], action) => {
+  switch (action.type) {
+    case types.LIKE_COMMENT:
+      return state.map((comment, idx) => {
+        if (idx === Number(action.key)) {
+          return {
+            ...comment,
+            likesCount: comment.liked ? comment.likesCount - 1 : comment.likesCount + 1,
```

Comment:
Please add necessary comments here for easy comprehension of the logic

Reply:
Ok. I have added the relevant comments. Thanks.